docs/running-on-yarn.md

```diff
@@ -514,6 +514,16 @@ To use a custom metrics.properties for the application master and executors, upd
   </td>
   <td>1.6.0</td>
 </tr>
+<tr>
+  <td><code>spark.yarn.executor.bindAddress</code></td>
+  <td><code>(executor hostname)</code></td>
+  <td>
+  Hostname or IP address where to bind listening sockets in YARN cluster and client mode.
```

Comment:
this seems odd to me.  The only thing this can really be set to in a multinode system is localhost or 0.0.0.0, right?  Otherwise its not like you can tell it to be node1.foo.com, node2.foo.com  or like individual ips right?  At least not by the user directly.

Reply:
For yarn running in k8s with Istio use case, the executor need to bind to 0.0.0.0

Reply:
I've updated the config to accept either `HOSTNAME` or `ALL_IPS`. Thanks for your feedback